db-service/lib/cql-functions.js

```diff
@@ -5,7 +5,11 @@ const StandardFunctions = {
   // length     : (x) => `length(${x})`,
   average: x => `avg(${x})`,
   search: function (ref, arg) {
-    if (!('val' in arg)) throw `SQLite only supports single value arguments for $search`
+    if ('xpr' in arg) {
+      let xpr = ''
+      for (const each of arg.xpr) xpr += typeof each === 'string' ? ` ${each} ` : this.search(ref, each)
```

Comment:
Don't we need an 'and' for strings?

Reply:
`$search=foo bar` (= `$search=foo AND bar`)
--> `xpr = [{ val: 'foo' }, 'and', { val: 'bar' }]`
--> sql snippet `x = ? and x = ?`, where `x = ?` is the result of `this.search(ref, each)`